content/en/docs/tutorials/cluster-management/kubelet-standalone.md

```diff
@@ -0,0 +1,526 @@
+---
+title: Running Kubelet in Standalone Mode
+content_type: tutorial
+weight: 10
+---
+
+<!-- overview -->
+
+This tutorial shows you how to run a standalone Kubelet instance.
+
+You may have have different motivations for running a standalone kubelet.
+This tutorial is aimed at introducing you to Kubernetes, even if you don't have
+much experience with it. You can follow this tutorial and learn about node setup,
+basic (static) Pods, and how Kubernetes manages containers.
+
+Once you have followed this tutorial, you could try using a cluster that has a
+{{< glossary_tooltip text="control plane" term_id="control-plane" >}} to manage pods
+and nodes, and other types of objects. For example,
+[Hello, minikube](/docs/tutorials/hello-minikube/).
+
+You can also run the kubelet in standalone mode to suit production use cases, such as
+to run the control plane for a highly available, resiliently deployed cluster. This
+tutorial does not cover the details you need for running a resilient control plane.
+
+## {{% heading "objectives" %}}
+
+* Install `cri-o`, and `kubelet` on a Linux system and run them as `systemd` services.
+* Launch a Pod running `nginx` that listens to requests on TCP port 80 on the Pod's IP address.
+* Learn how the different components of the solution interact among themselves.
+
+{{< caution >}}
+The Kubelet configuration used for this tutorial is insecure by design and should
+_not_ be used in a production environment.
```

Comment:
This contradicts with the statement on line 21.

Reply:
Can you elaborate?

Reply:
on line 21,

> You can also run the kubelet in standalone mode to suit production use cases, such as
to run the control plane for a highly available, resiliently deployed cluster.

Now this statement says that the configuration is not suitable for production use cases.

Reply:
It doesn't say that they can use the same Kubelet configuration shown in the tutorial for a production cluster. At least I don't see it. The last sentence of the same paragraph says: "This tutorial does not cover the details you need for running a resilient control plane".

Reply:
I've also added various `# Do NOT use in production clusters!` notes to the Kubelet's config file, so it makes it clear that what is shown in the tutorial should not be used in a production environment.